.github/workflows/tikv-clippy-darwin-amd64.yml

```diff
@@ -0,0 +1,78 @@
+name: Clippy (Darwin AMD64)
+
+env:
+  CMAKE_VERSION: "3.28.0"
+  CMAKE_URL: "https://github.com/Kitware/CMake/releases/download/v3.28.0/cmake-3.28.0-macos-universal.tar.gz"
```

Comment:
it seems there is no need set the url, especially it should depends on `CMAKE_VERSION` env var.